apps/backend/minecraft_manager/config.py

```
@@ -21,6 +21,8 @@ class Settings:
     backup_root: Path = Path("/data/backups/coordinated")
     auth_mode: str = "local"
     auth_cookie_secure: bool = True
+    host_agent_url: str = ""
+    host_agent_token_file: str = "/run/secrets/host_agent_token"
```

Ruff (0.16.2)

[error] Possible hardcoded password assigned to: "host_agent_token_file"

(S105)